ui-admin/src/components/InfoCard.tsx

```diff
@@ -47,7 +47,7 @@ import { isEmpty } from 'lodash'
  * ```
  */
 export function InfoCard({ children }: { children: React.ReactNode }) {
-  return <div className="card w-75 border shadow-sm mb-3">
+  return <div className="card w-100 border shadow-sm mb-3">
```

Comment:
nice- in the portal dashboard redesign PR i hedged here and kept 75 as the default, allowing overrides. but i think you're right to just make it 100